### PR TITLE
Update PrivatGames

### PR DESCRIPTION
**Pull request by DavidFrings**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [x] Remove directory of a server
- [x] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/privatgames/manifest.json [hunk 1]
```diff
@@ -11,8 +14,25 @@
...
+  "gamemodes": {
+    "project-1": {
+      "name": "Slowrun",
+      "command": "/server project-1"
+    },
+    "project-2": {
+      "name": "Survival",
+      "command": "/server project-2"
+    },
+    "project-3": {
```
Comment (DoJapHD) [inline minecraft_servers/privatgames/manifest.json:29]: If nonexistent yet, remove!

File: minecraft_servers/privatgames/manifest.json
Comment (DoJapHD) [inline minecraft_servers/privatgames/manifest.json:1]: Server is not reachable

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/privatgames/manifest.json
Comment (DavidFrings) [reply minecraft_servers/privatgames/manifest.json:1]: if you try to connect, with your client you can!

**Comment by DavidFrings** [comment]
Online Prove:
https://imgur.com/vykUziJ

**Comment by DavidFrings** [comment]
Mybe the bot has Problems because the server is ipv6 only .)

**Comment by DoJapHD** [comment]
https://i.imgur.com/1G78wTe.png

**Comment by DavidFrings** [comment]
ok, thats wierd. Do you have ipv6? or have you disabled cloudflare reverseproxys?

**Comment by DoJapHD** [comment]
The domain has no entry linking to any Minecraft server. Are you sure that you didn't just mess with your hosts file?

**Comment by DavidFrings** [comment]
Ok, I see that I have dns Issues. I am closing this pull request until i have fix all my issues and open a new request then. THX for your help :)

**Comment by DavidFrings** [comment]
I created an issue in Velocity, and they were able to help me fix my problem. My issue was that I had proxying activated in Cloudflare, but it should have been set to DNS only.

I'll make a new commit shortly. 

:)

**Pull request closed without merging**